packages/block-library/src/gallery/block.json

```diff
@@ -3,56 +3,19 @@
 	"name": "core/gallery",
 	"category": "media",
 	"attributes": {
-		"images": {
+		"imageUploads": {
```

Comment:
What is this attribute for? I was expecting to see no attributes referring to the images since it's not in inner blocks?

Reply:
When transforming from a drag and drop of  images onto an insertion point within the gallery we needed a way to capture those files and process them, similar to the shortCode transform, once images are processed and transformed to innerBlocks this attribute is removed and is never saved in the block content.

Reply:
Actually, the image block already has the same behavior, basically if you create an image block with a local url, it will first upload the image on mount which means, this might not be necessary.

Reply:
Essentially it is just handing the local url over to the image block and the image block deals with it. I think there was some issue with getting the local url down to the image block in the context of the transform, which is why it is being passed through the temporary attribute, but I will take another look at it next week and clarify that as it was a long time ago that I look at this.

Reply:
I think we already have a few transforms that work that way. Let me try to find one.

Reply:
This one for instance https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/image/transforms.js#L133-L141

Reply:
on looking at it again I remember how I ended up at this. I was initially handling the image processing in the gallery, hence the need for the temp attribute, but then realised we can just offload all image handling to the image block, which the gallery block does, but didn't then make the connection to do this right back at the transform level ... thanks for questioning this as it was an obvious oversight - file transform is now fully handled in the transform.